web/flat-web/src/apiMiddleware/Rtm.ts

```diff
@@ -2,7 +2,7 @@ import AgoraRTM, { RtmChannel, RtmClient } from "agora-rtm-sdk";
 import polly from "polly-js";
```

Comment:
It has nothing to do with this PR.

Reply:
should i put rtm in another pr or rename this pr's title?